bbc/simorgh
# Create image placeholder

Part of #449 

_This PR creates an image placeholder component as the wrapper now acts as a placeholder with the padding bottom defined by the image's ratio meaning it is fully responsive. It also creates a base64 SVG for the BBC blocks in the image placeholder. Add stories and tests for the Image placeholder._

Update: the integration of this component will be handled in another PR.  


## Test instructions 
To see the placeholder rendering at various ratio's and being fully responsive visit storybook with this branch checked out. This should be tested across different browsers.
